fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/BaseDataServletComponentSpec.groovy

```diff
@@ -206,23 +203,22 @@ abstract class BaseDataServletComponentSpec extends Specification {
      */
     Response makeAbstractRequest(Closure queryParams=this.&getQueryParams) {
         // Set target of call
-        def httpCall = jtb.getHarness().target(getTarget())
+        def httpCall = jtb.harness.target(target)
```

Comment:
You are indeed correct, and in fact it's called out in the language spec: http://docs.groovy-lang.org/latest/html/documentation/#properties

And, if you think about it, it's required, otherwise there'd be a real possibility of an infinite loop! (or requiring something special if someone build a dumb getter).

I'll make a note of not doing this in the future, and I think Code Narc (Groovy's checkstyle equivalent) has rules to prevent this. Thx for the eagle-eye!